packages/core/src/tracing/trace.ts

```diff
@@ -34,14 +34,14 @@ export function trace<T>(
 
   const parentSpan = scope.getSpan();
 
-  function startActiveSpan(): Span | undefined {
+  function getChildSpanOrTransaction(): Span | undefined {
```

Comment:
l: maybe
```suggestion
  function createChildSpanOrTransaction(): Span | undefined {
```

`get` implies we are not creating anything, kind of?